src/installer/pkg/sfx/Microsoft.NETCore.App/Directory.Build.props

```diff
@@ -115,6 +115,8 @@
     <PlatformManifestFileEntry Condition="'$(PgoInstrument)' != ''" Include="clrjit.pgd" IsNative="true" />
     <PlatformManifestFileEntry Include="libclrjit.so" IsNative="true" />
     <PlatformManifestFileEntry Include="libclrjit.dylib" IsNative="true" />
+    <PlatformManifestFileEntry Include="libclrinterpreter.so" IsNative="true" />
```

Comment:
We build the clrinterpreter only for x64/arm64 Checked/Debug builds. Does this need to be conditional?